src/mcts/search.cc

```diff
@@ -818,48 +847,62 @@ int SearchWorker::PrefetchIntoCache(Node* node, int budget) {
 // 4. Run NN computation.
 // ~~~~~~~~~~~~~~~~~~~~~~
 void SearchWorker::RunNNComputation() {
-  // This function is so small as to be silly, but its parent function is
-  // conceptually cleaner for it.
-  if (computation_->GetBatchSize() != 0) computation_->ComputeBlocking();
+  // Do not do NN eval if there's nothing to compute, or if it's an out-of-order
+  // eval, don't compute.
```

Comment:
Surely this comment can be simplified :)

Reply:
Done. :)